tiflash/tiflash-configuration.md

```diff
@@ -178,13 +178,19 @@ delta_index_cache_size = 0
     ## see known issue [#5576](https://github.com/pingcap/tiflash/issues/5576).
     # dt_enable_logical_split = false
 
-    ## The memory usage limit for the generated intermediate data when a single
-    ## coprocessor query is executed. The default value is 0, which means no limit.
+    ## The memory usage limit for the generated intermediate data in a single query.
+    ## When the value is an integer, the unit is byte. For example, 34359738368 means 32 GiB of memory limit, and 0 means no limit.
+    ## When the value is a floating-point number in the range of [0.0, 1.0), it means the ratio of the allowed memory usage to the total memory of the node. For example, 0.8 means 80% of the total memory, and 0.0 means no limit.
+    ## The default value is 0, which means no limit.
+    ## When a query attempts to consume memory that exceeds the limit of this parameter, the query is terminated and an error is reported.
     max_memory_usage = 0
 
-    ## The memory usage limit for the generated intermediate data when all queries
-    ## are executed. The default value is 0 (in bytes), which means no limit.
-    max_memory_usage_for_all_queries = 0
+    ## The memory usage limit for the generated intermediate data in all queries.
+    ## When the value is an integer, the unit is byte. For example, 34359738368 means 32 GiB of memory limit, and 0 means no limit.
+    ## When the value is a floating-point number in the range of [0.0, 1.0), it means the ratio of the allowed memory usage to the total memory of the node. For example, 0.8 means 80% of the total memory, and 0.0 means no limit.
+    ## The default value is 0.8, which means 80% of the total memory.
+    ## When the queries attempt to consume memory that exceeds the limit of this parameter, the queries are terminated and an error is reported.
```

Comment:
```suggestion
    ## When a query attempts to consume memory that exceeds the limit, the query is terminated and an error is reported.
```

Reply:
@Oreoxmt Since this para is to limit the memory usage of all queries, `a query` is not that accurate here.